controllers/articles.controller.js

```diff
@@ -17,3 +17,11 @@ exports.getAllArticles = (request, response, next) => {
   .catch(next);
 };
 
+exports.getAllCommentsByArticleId = (request, response, next) => {
+  const articleId = request.params.article_id;
+  selectAllCommentsByArticleId(articleId)
+  .then((comments) => {
+    response.status(200).send({comments: comments});
```

Comment:
If you're making an object with a key of comments with a value of comments this can be shortened to ```{comments}``` this will achieve the same result.